tests/ChatClient.hs

```diff
@@ -191,7 +192,10 @@ groupLinkViaContactVRange = mkVersionRange 1 2
 createTestChat :: FilePath -> ChatConfig -> ChatOpts -> String -> Profile -> IO TestCC
 createTestChat tmp cfg opts@ChatOpts {coreOptions = CoreChatOpts {dbKey}} dbPrefix profile = do
   Right db@ChatDatabase {chatStore} <- createChatDatabase (tmp </> dbPrefix) dbKey False MCError
-  Right user <- withTransaction chatStore $ \db' -> runExceptT $ createUserRecord db' (AgentUserId 1) profile True
+  Right user <- withTransaction chatStore $ \db' -> runExceptT $ do
+    user <- createUserRecord db' (AgentUserId 1) profile True
+    createNoteFolder db' user
+    pure user
```

Comment:
probably better to revert, this would cause multiple tests to fail and a lot of noise to fix them